mnelab/mainwindow.py

```diff
@@ -8,29 +8,35 @@
 from os.path import isfile, isdir
 from functools import partial
 import numpy as np
-
 import mne
+import qtpy
+import matplotlib
+matplotlib.use("Qt5Agg")
```

Comment:
Did you check if these three lines are really necessary? It seems like they aren't.

Reply:
qtpy is not necessary, but the backend of matplotlib needs to be set